census/src/main/scala/grasshopper/census/search/AddressInterpolator.scala

```diff
@@ -91,12 +59,48 @@ object AddressInterpolator {
     Feature(schema, values)
   }
 
+  private def prefix(f: Feature, a: Int): String = {
+    val addressIsEven = a % 2 == 0
+    val rightRangeIsEven: Boolean = rangeIsEven(f, false)
+
+    val leftRangeIsEven: Boolean = rangeIsEven(f, true)
+
+    if (addressIsEven && rightRangeIsEven)
+      "R"
+    else if (addressIsEven && leftRangeIsEven)
+      "L"
+    else if (!addressIsEven && !rightRangeIsEven)
+      "R"
+    else if (!addressIsEven && !leftRangeIsEven)
+      "L"
+    else
+      "R"
+  }
+
+  private def rangeIsEven(f: Feature, isLeft: Boolean): Boolean = {
+    val prefix = if (isLeft) "L" else "R"
+    val fromEven = f.values.getOrElse(s"${prefix}FROMHN", 0)
+    val fromRange = fromEven match {
+      case "" => 0
```

Comment:
Is there a legit reason for the `FROMHN`/`TOHN` properties to be present, but set to empty string?  If no, seems like another spot to log the details of the `Feature` so we can track down where/why we have bad/missing data.